app/src/androidTest/java/com/reach5/identity/sdk/demo/MainActivityTest.kt

```diff
@@ -58,6 +59,38 @@ class MainActivityTest {
         instantiateReachFiveClient("", CLIENT_ID)
     }
 
+    @Test
+    fun testSuccessfulClientConfigFetch() = clientTest(initialize = false) { client ->
```

Comment:
Please see the function definition.

I like this because we have a common test bootstrap. If you like it too I can apply it to the other tests in a separate PR.

Reply:
The default is to initialize the client. So a regular test looks like:
```
@Test
fun myTest() = clientTest { client ->
    assert(iLikeToRefactor, true)
}
```

Reply:
We can rename `clientTest` of course, to just `test` since this whole file is about client testing.

Reply:
Ok to use it everywhere. I prefer the `clientTest` term, "test" can mean anything.